app/models/parachute.rb

```diff
@@ -0,0 +1,2 @@
+class Parachute < ApplicationRecord
+end
```

Comment:
- [ ] Kindly, add some validation to this model.  e.g. lUniqueness, required presence, etc.

That way, your database integrity will be preserved and wrong data won't be entered. 👍